resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -39,6 +39,7 @@ public class CircuitBreakerConfig implements Serializable {
 
     private static final long serialVersionUID = -5429814941777001669L;
 
+    public static final boolean DEFAULT_IGNORE_UNKNOWN_EXCEPTIONS_ENABLED = false;
```

Comment:
Let's move this completly out of the CircuitBreakerConfig. It's not needed anymore, correct?

Reply:
Correct, I must have missed it in my previous commit. Will remove it.